cmd/root.go

```diff
@@ -177,6 +184,18 @@ func parseConfig(conf *proxy.Config, args []string) error {
 	return nil
 }
 
+type cmdLogger struct {
+	*cobra.Command
+}
+
+func (c cmdLogger) Log(msg string, data ...interface{}) {
+	c.Printf(msg, data...)
+}
+
+func (c cmdLogger) Error(msg string, data ...interface{}) {
+	c.PrintErrf(msg, data...)
+}
```

Comment:
I'm not sure I follow why we are adding this now.

Reply:
Removed this in favor of logging in parseConfig.